subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeMultiProjectIntegrationTest.groovy

```diff
@@ -16,24 +16,24 @@
 
 package org.gradle.ide.xcode
 
-import org.gradle.ide.xcode.fixtures.XcodeWorkspacePackage
-import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.ide.xcode.fixtures.AbstractXcodeIntegrationSpec
+import org.gradle.nativeplatform.fixtures.app.CppHelloWorldApp
+import org.gradle.nativeplatform.fixtures.app.ExeWithLibraryUsingSwiftLibraryHelloWorldApp
 import org.gradle.nativeplatform.fixtures.app.SwiftHelloWorldApp
+import org.gradle.util.CollectionUtils
 
-class XcodeMultiProjectIntegrationTest extends AbstractIntegrationSpec {
-    private static final String WORKSPACE_NAME = "workspace"
-    def app = new SwiftHelloWorldApp()
+import static org.gradle.ide.xcode.internal.XcodeUtils.toSpaceSeparatedList
+
+class XcodeMultiProjectIntegrationTest extends AbstractXcodeIntegrationSpec {
```

Comment:
the single project tests are separated by language, whereas they are all grouped into a single class for multi-project tests. I would separate the multi-projects by language. Do this as a later change so we can get the current changes into mater.